engine/lib/phx/src/system/guid.rs

```diff
@@ -2,17 +2,21 @@ use std::sync::atomic::{AtomicU64, Ordering};
 
 static NEXT_ID: AtomicU64 = AtomicU64::new(1);
 
-#[no_mangle]
-pub extern "C" fn GUID_Create() -> u64 {
-    NEXT_ID.fetch_add(1, Ordering::Relaxed)
+#[repr(C)]
+pub struct GUID {
```

Comment:
I'd call it `Guid` to follow Rust coding standard. You can use `name = "GUID"` parameter of the macro to rename it for Lua:
```rust
#[luajit_ffi_gen::luajit_ffi(name = "GUID")]
```

Also brackets should be on the same line as struct itself if there are no fields. If you run `cargo fmt` it will be done automatically. I personally just set 'Format on save' parameter in vscode to call it automatically.

Reply:
I didn't find any use of GUID in the Rust code, so proceeded with the suggested renaming.